### DISCUSSION TITLE
archery查询后导出文件,最多只能导出5000条,有办法解决吗

### DISCUSSION THREAD

**Discussion by 18310712834** [hhyo/Archery#2336] [General]
archery查询后导出文件,最多只能导出5000条,有办法解决吗

**Comment by cxheny** [comment]
可以调大查询限制。

**Comment by lanheader** [comment]
<img width="1057" alt="image" src="https://github.com/hhyo/Archery/assets/52522047/4a3d943f-2544-44c5-bc6f-a4218339dddc">
<img width="1558" alt="image" src="https://github.com/hhyo/Archery/assets/52522047/84dd93c4-845d-44bc-b8fa-7496c1629938">
可以这样修改

**Reply by 18310712834** [reply]
非常感谢
